imcsdk/imccoreutils.py

```diff
@@ -285,10 +285,17 @@ def extract_molist_from_method_response(method_response,
     """
 
     mo_list = []
-    if len(method_response.out_configs.child) == 0:
-        return mo_list
+    if hasattr(method_response, "out_config"):
+        if len(method_response.out_config.child) == 0:
+            return mo_list
+    else:
+        if len(method_response.out_configs.child) == 0:
```

Comment:
@vvb Yes in IMC we have "configResolveClass","configResolveChildren" and "configResolveDn".
We don't have configResolveClasses and configResolveDns in IMC.

**configResolveDn** returns "**outConfig**"

<configResolveDn cookie="0000786501/54d31b55-00b7-10b7-838d-0d8fd9f6083c" response="yes" dn="sys/rack-unit-1">**<outConfig>**<computeRackUnit dn="sys/rack-unit-1" adminPower="policy" availableMemory="131072" model="UCSC-C240-M3L" memorySpeed="1600" name="UCS C240 M3L" numOfAdaptors="1" numOfCores="16" numOfCoresEnabled="16" numOfCpus="2" numOfEthHostIfs="2" numOfFcHostIfs="2" numOfThreads="32" operPower="on" originalUuid="4A42480B-345D-4CD0-8824-A399795E279C" presence="equipped" serverId="1" serial="FCH1752V07Z" totalMemory="131072" usrLbl="hffgfg" uuid="4A42480B-345D-4CD0-8824-A399795E279C" vendor="Cisco Systems Inc" cimcResetReason="watchdog-reset
" ></computeRackUnit></outConfig></configResolveDn>


Whereas **configResolveClass** and **configResolveChildren** returns "**outConfigs**"

<configResolveClass cookie="0000786380/d1a6d943-00b7-10b7-838c-0d8fd9f6083c" response="yes" classId="computeRackUnit">**<outConfigs>**<computeRackUnit dn="sys/rack-unit-1" adminPower="policy" availableMemory="131072" model="UCSC-C240-M3L" memorySpeed="1600" name="UCS C240 M3L" numOfAdaptors="1" numOfCores="16" numOfCoresEnabled="16" numOfCpus="2" numOfEthHostIfs="2" numOfFcHostIfs="2" numOfThreads="32" operPower="on" originalUuid="4A42480B-345D-4CD0-8824-A399795E279C" presence="equipped" serverId="1" serial="FCH1752V07Z" totalMemory="131072" usrLbl="hffgfg" uuid="4A42480B-345D-4CD0-8824-A399795E279C" vendor="Cisco Systems Inc" cimcResetReason="watchdog-reset" ></computeRackUnit></outConfigs></configResolveClass>